src/actions/acquire_stepped_freq_tdomain_iq.py

```diff
@@ -119,10 +119,24 @@ def __call__(self, schedule_entry_name, task_id):
         for recording_id, measurement_params in enumerate(
             self.measurement_params_list, start=1
         ):
+            start_time = utils.get_datetime_str_now()
             data = self.acquire_data(measurement_params, task_id)
-            sigmf_md = self.build_sigmf_md(task_id, measurement_params, data)
+            end_time = utils.get_datetime_str_now()
```

Comment:
Same comment as for the M4S action - time may not be better pushed to the `radio_interface` for a more exact time.